Code/application/Gui/Image/ColormapGpuProgram.cpp

```diff
@@ -0,0 +1,39 @@
+#include "ColormapGpuProgram.h"
```

Comment:
Add copyright header. Also check all other new files.

/*
 * The information in this file is
 * Copyright(c) 2020 Ball Aerospace & Technologies Corporation
 * and is subject to the terms and conditions of the
 * GNU Lesser General Public License Version 2.1
 * The license text is available from   
 * http://www.gnu.org/licenses/lgpl.html
 */